### PR TITLE
'ut-run changelog' initial implementation

### PR DESCRIPTION
**Pull request by stambata**
ut-run bin command

### CODE REVIEW

File: bin/ut-run.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const pkgJson = require('../package.json');
```
Comment (kalinkrustev) [inline bin/ut-run.js:1]: The first row must be #!/usr/bin/env node
Also make sure the file has executable flag.
```sh
git update-index --chmod=+x bin/ut-run.js
```

File: package.json [hunk 2]
```diff
@@ -15,29 +17,30 @@
...
   },
   "peerDependencies": {
-    "why-is-node-running": "^2.2.0",
-    "json-schema-to-markdown": "1.1.1"
+    "json-schema-to-markdown": "1.1.1",
+    "why-is-node-running": "^2.2.0"
   },
   "bin": {
-    "ut-doc": "./bin/ut-doc.js"
+    "ut-doc": "./bin/ut-doc.js",
```
Comment (kalinkrustev) [inline package.json:42]: Let's remove ut-doc from here and also the file.

**Review by kalinkrustev — CHANGES_REQUESTED** [review]

File: bin/ut-run.js (hunk 1 above)
Comment (stambata) [reply bin/ut-run.js:3]: [done](https://github.com/softwaregroup-bg/ut-run/pull/100/commits/ff1dbfb32c4a3b6299cbaea18b11209b82682250)

File: package.json (hunk 2 above)
Comment (stambata) [reply package.json]: [done](https://github.com/softwaregroup-bg/ut-run/pull/100/commits/ff1dbfb32c4a3b6299cbaea18b11209b82682250)

**Review by kalinkrustev — APPROVED** [review]

**Pull request merged**